packages/app/src/app/pages/Sandbox/Editor/Workspace/Files/index.js

```diff
@@ -17,6 +17,11 @@ class Files extends React.Component {
     this.directory.onCreateDirectoryClick();
   };
 
+  uploadFile = () => {
+    // INCREDIBLY BAD PRACTICE! TODO: FIX THIS
```

Comment:
Hahaha nice that you even copied this over! Let's keep that for now 😄

Reply:
:) haha. Fix in the future